Bits Code: add GitLab support (#37558)
DataDog/documentation: content/en/bits_ai/bits_ai_dev_agent/_index.md

## Overview

Bits Code is a generative AI coding assistant that uses Datadog observability data to automatically diagnose and fix issues in your code. It integrates with [source code providers](#supported-source-code-providers) to create production-ready pull/merge requests, then iterates on changes using CI logs and developer feedback.

{{< img src="bits_ai/dev_agent/sessions_overview.png" alt="A tab titled 'Sessions' shows a text field with suggestions underneath" style="width:100%;" >}}

Each time Bits Code investigates an issue or generates a fix, it creates a [session](#sessions), which captures the agent's analysis, actions, and any resulting code changes across supported Datadog products. Set up [automations][28] to have Bits Code run sessions on a schedule or in response to signals from other Datadog products, such as a new APM Recommendation or flaky test.

> To get started with Bits Code, [set up the GitHub or GitLab integration][6] and complete any additional configuration. Then, [start your first session](#start-a-session).

Author: ```suggestion
To get started with Bits Code, [set up a source code integration][6] and complete any additional configuration. Then, [start your first session](#start-a-session).
```
